api/v1/openapi.yaml

```diff
@@ -2875,6 +2878,24 @@ definitions:
       users:
         description: number of users of this selector in the cache
         type: integer
+  SRv6:
```

Comment:
Let's change this to `Srv6`. I'm sure it's weird, but otherwise, it results in an even stranger file name, `s_rv6.go`.

Reply:
Sure @YutaroHayakawa, let me change that.

Reply:
Thanks!